lib/server.go

```diff
@@ -2069,20 +2053,8 @@ func (srv *Server) _handleBlock(pp *Peer, blk *MsgDeSoBlock) {
 	glog.Infof(CLog(Cyan, fmt.Sprintf("Server._handleBlock: Received block ( %v / %v ) from Peer %v",
 		blk.Header.Height, srv.blockchain.headerTip().Height, pp)))
 
-	// If the node is running a Fast-HotStuff validator and the consensus is running,
```

Comment:
The check to determine whether to pass a block to the FastHotStuffConsensus doesn't need to shortcut all of Server._handleBlock. It is just one special case out of multiple options to process the block depending on our syncing status